### DISCUSSION TITLE
copilot svc commands without the need of the copilot folder locally.

### DISCUSSION THREAD

**Discussion by ejayesh** [aws/copilot-cli#4545] [Ideas]
Like the copilot app command, it would be useful to run the copilot svc command without the need of having a copy of the copliot folder locally.

At present we are having to checkout multiple repositories with our copilot config just so we can use the svc commands like copilot svc exec, status, logs.

**Comment by bvtujo** [comment]
Hey @ejayesh, the copilot folder and .workspace file in particular are a kind of shorthand for linking a workspace to an app, equivalent to passing in the `--app (-a)` flag for all your copilot commands. You can override the registered workspace, or specify an app where a workspace doesn't exist, by passing the `app` flag to operational commands, or the `--name` flag to app commands like `app show`.

```sh
copilot -a my-app svc logs -n frontend
copilot app show --name my-app
copilot env show -a my-app
```

Does that help?

**Reply by ejayesh** [reply]
Ah yes, this is the sort of thing I was looking for.  We have multiple AWS accounts and we will have dozens of applications in each account.  From an operations point of view we wanted support staff to be able to see/mange what is deployed in each account without having the .workspace locally.  

So the pattern for us would be, login to AWS with.
aws sso login --profile xxx
copilot app ls  (to list the apps in the account)
copilot app show --name app_name   (to list services)
copilot -a app_name svc exec svc_name    (to shell into the container)

This works for us.  Many thanks.
